components/browser/state/src/test/java/mozilla/components/browser/state/state/TabPartitionTest.kt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class TabPartitionTest {
+
+    @Test
+    fun `GIVEN a null tab partition THEN tab partition is empty`() {
+        val tabPartition: TabPartition? = null
+
+        assertEquals(tabPartition.isEmpty(), true)
+        assertEquals(tabPartition.isNotEmpty(), false)
+    }
+
+    @Test
+    fun `GIVEN a tab partition with no tab group THEN tab partition is empty`() {
+        val tabPartition = TabPartition("test")
+
+        assertEquals(tabPartition.isEmpty(), true)
+        assertEquals(tabPartition.isNotEmpty(), false)
+    }
+
+    @Test
+    fun `GIVEN a tab partition with empty tab groups THEN tab partition is empty`() {
+        val tabPartition = TabPartition("test", listOf(TabGroup(), TabGroup()))
+
+        assertEquals(tabPartition.isEmpty(), true)
+        assertEquals(tabPartition.isNotEmpty(), false)
+    }
+
+    @Test
+    fun `GIVEN a tab partition with non-empty tab group THEN tab partition is not empty`() {
+        val tabPartition = TabPartition("test", listOf(TabGroup("test", "test", listOf("tab1"))))
+
+        assertEquals(tabPartition.isEmpty(), false)
+        assertEquals(tabPartition.isNotEmpty(), true)
+    }
+
+    @Test
+    fun `GIVEN a tab partition with non-empty tab group THEN get group by name will return the group`() {
+        val tabPartition = TabPartition("test", listOf(TabGroup("test id", "abc", listOf("tab1", "tab2"))))
+
+        assert(tabPartition.getGroupByName("abc") != null)
+        assertEquals(listOf("tab1", "tab2"), tabPartition.getGroupByName("abc")?.tabIds)
+        assertEquals(tabPartition.isEmpty(), false)
+        assertEquals(tabPartition.isNotEmpty(), true)
+    }
+
+    @Test
+    fun `GIVEN a tab partition with non-empty tab group THEN get group by ID will return the group`() {
+        val tabPartition = TabPartition("test", listOf(TabGroup("test id", "abc", listOf("tab1", "tab2"))))
+
+        assert(tabPartition.getGroupById("test id") != null)
```

Comment:
That's a JDK `assert` not a junit one. `assertTrue`.

Reply:
Thanks for catching these.  I'll update.